src/components/POLine/POLineVersionView/DonorsVersionView/DonorsVersionView.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import {
+  useCallback,
+  useContext,
+  useMemo,
+} from 'react';
+
+import {
+  DonorsListContainer,
+  VersionViewContext,
+} from '@folio/stripes-acq-components';
+import { TextLink } from '@folio/stripes/components';
+
+import { getDonorUrl } from './utils';
+
+export const DonorsVersionView = ({ version }) => {
+  const donorOrganizationIds = version?.donorOrganizationIds;
+  const versionContext = useContext(VersionViewContext);
+
+  const changedDonorIds = useMemo(() => {
+    const changedDonors = versionContext?.changes?.filter(({ path, type }) => {
+      return path.startsWith('donorOrganizationIds') && type === 'create';
+    });
+    const donorIds = changedDonors.map(({ values }) => values.filter(Boolean));
+
+    return donorIds.flat();
+  }, [versionContext?.changes]);
+
+  const renderKeyValueComponent = useCallback(({ value, id, asLink = false }) => {
```

Comment:
I doubt whether the name of the function corresponds to its contents

Reply:
updated